src/Messages.sol

```diff
@@ -96,25 +96,55 @@ library ConnectorMessages {
      * 0: call type (uint8 = 1 byte)
      * 1-8: poolId (uint64 = 8 bytes)
      * 9-25: trancheId (16 bytes)
-     * 26-46: user (Ethereum address, 20 bytes)
-     * 47-78: validUntil (uint256 = 32 bytes)
+     * 25-45: user (Ethereum address, 20 bytes - Skip 12 bytes from 32-byte addresses)
+     * 57-65: validUntil (uint64 = 8 bytes)
      * 
      * TODO: use bytes32 for user (for non-EVM compatibility)
```

Comment:
@offerijns I started working on this but we would need to also update `Transfer` and the storage items storing `address` now. I prefer to do that in a follow up PR unless if you think we shouldn't do this now?

Reply:
I think that's totally fine in a separate PR!

Reply:
Yah separating issues into separate PRs is a good idea. I'll note here so I don't forget though, it looks like we're double-reading a byte here. Tranche ID is 9-25, while user is 25-45.

Reply:
@AStox I first thought this as well but it doesn't seem to be the case:

``` solidity
  poolId = uint64(_msg.indexUint(1, 8));
  trancheId = bytes16(_msg.index(9, 16));
  user = address(bytes20(_msg.index(25, 20)));
  validUntil = uint64(_msg.indexUint(57, 8));
```

So, `poolId` would be `1 + 8 = 9`, and then we would be reading `trancheId` from byte at index `9`, which would be another example of double-reading.

The docs were wrong indeed tho, as it's `9-24`. Thanks for the catch!